src/routes/user/userComponents/userData.svelte

```diff
@@ -26,35 +168,126 @@
+                                    type="text"
+                                    name="inputField"
+                                    placeholder={personalData.phoneNumber}
+                                    bind:this={phonenumberReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left"> Straße: </th>
+                    <td>
+                        <form id="form">
+                            <label class="label">
+                                <input
+                                    disabled={!formEditEnabled}
+                                    class="form--disabled input variant-form-material"
+                                    type="text"
+                                    name="inputField"
+                                    placeholder={personalData.street}
+                                    bind:this={streetReference}
                                 />
                             </label>
                         </form>
                     </td>
                 </tr>
+                <tr>
+                    <th class="text-left"> Hausnummer: </th>
+                    <td>
+                        <form id="form">
+                            <label class="label">
+                                <input
+                                    disabled={!formEditEnabled}
+                                    class="form--disabled input variant-form-material"
+                                    type="text"
+                                    name="inputField"
+                                    placeholder={personalData.houseNumber}
+                                    bind:this={houseNumberReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left">
+                        <span class="mr-5">Ort:</span>
+                    </th>
+                    <td>
+                        <form id="form">
+                            <label class="label">
+                                <input
+                                    disabled={!formEditEnabled}
+                                    class="form--disabled input variant-form-material"
+                                    type="text"
+                                    name="inputField"
+                                    placeholder={personalData.city}
+                                    bind:this={cityReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left"> Postleitzahl: </th>
```

Comment:
```suggestion
                    <th class="text-left">Postleitzahl:</th>
```
Oder waren die Spaces da mit Absicht?